store/people.json

```diff
@@ -0,0 +1 @@
+[{"type":"Teacher","name":"walid","id":455,"age":12,"specialization":null},{"type":"Teacher","name":"walid","id":55,"age":12,"specialization":"walid"}]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/203615337-b4b1ffbb-3a31-41ca-bcb3-4638feb2e7a3.png)

- [ ] When i deleted the people.json file and then i create a student and a teacher and quit the application then your application showed an error shown in the image above :arrow_up: Kindly fix this error :+1: and see all the cases like if file doesn't exist and if file exist but empty :thinking: